rero_ils/modules/items/jsonschemas/items/item-v0.0.1.json

```diff
@@ -203,7 +203,7 @@
         },
         "received_date": {
           "format": "date",
-          "title": "Date of reception",
+          "title": "Date of receipt",
```

Comment:
How this is correct I don't know.

Reply:
*Receipt* is a librarian term associated to the reception of a journal issue.